jupyterlab/staging/package.json

```diff
@@ -71,8 +71,7 @@
     "@jupyterlab/markdownviewer": "~4.0.0-alpha.19",
     "@jupyterlab/markdownviewer-extension": "~4.0.0-alpha.19",
     "@jupyterlab/markedparser-extension": "~4.0.0-alpha.19",
-    "@jupyterlab/mathjax2": "~4.0.0-alpha.19",
-    "@jupyterlab/mathjax2-extension": "~4.0.0-alpha.19",
+    "@jupyterlab/mathjax-extension": "~4.0.0-alpha.19",
```

Comment:
We should be able to revert the change made to `jupyterlab/staging/package.json` since this file is updated during the release process.

Reply:
This will help make this CI check pass: https://github.com/jupyterlab/jupyterlab/actions/runs/4189017035/jobs/7260844694